rest-service/manager_rest/cluster_status_manager.py

```diff
@@ -102,7 +102,7 @@
 }
 
 
-def get_cluster_status(detailed=False):
+def get_cluster_status(detailed=False) -> typing.Dict[str, typing.Any]:
```

Comment:
```
manager_rest/cluster_status_manager.py:122: error: Incompatible types in assignment (expression has type "str", target has type "Dict[Any, Dict[str, Any]]")
```
this isn't really on this line, and this type annotation is fiiiine, but it's about the `cluster_status['status'] = _get_overall_state(cluster_status)` line - mypy really doesn't like heterogenous dicts (and it's right, they're ugly)

I suggest doing this:
```python
def get_cluster_status(detailed=False) -> typing.Dict[str, typing.Any]:
    cluster_nodes, cloudify_version = _get_cluster_details()
    _add_monitoring_data(cluster_nodes)

    cluster_services = {
        CloudifyNodeType.BROKER:
        _get_broker_state(cluster_nodes, cloudify_version, detailed),
        CloudifyNodeType.DB:
        _get_db_state(cluster_nodes, cloudify_version, detailed),
        CloudifyNodeType.MANAGER:
        _get_manager_state(cluster_nodes, cloudify_version, detailed),
    }
    return {
        'services': cluster_services,
        'status': _get_overall_state(cluster_services)
    }
```
that way, there's no separate local variable for `cluster_status`, so just the typehint from the return value suggests that it's Any
also of course change `_get_overall_state` to take the services dict.

And I think it's a good change anyway, even though it's mostly just babysitting mypy